src/Events/ContentUpdating.php

```diff
@@ -20,9 +21,12 @@ class ContentUpdating
     public function __construct(public AbstractContentModel $page)
     {
         $page->updated_user_id = auth()->user() ? auth()->user()->id : $page->created_user_id;
-        if(!$page->is_draft && is_null($page->publish_at)) {
+        if (!$page->is_draft && is_null($page->publish_at)) {
             $page->publish_at = Carbon::now()->subMinute();
         }
 
```

Comment:
Please add/adjust tests that cover this functionality

Reply:
Done!